Entities/Clothing/Belt/belts.yml

```diff
@@ -505,6 +505,8 @@
           - SmokeOnTrigger
     sprite: Clothing/Belt/belt_overlay.rsi
   - type: Appearance
+  - type: ExplosionResistance
+      damageCoefficient: 0.1
```

Comment:
change this to be 0.5 so strong enough explosion still trigger them

Reply:
0.25 would be more appropriate for that, 0.5 will make it go off at pretty much any explosion iirc.

Reply:
'tis done my liege.